### DISCUSSION TITLE
Do you think it is a good idea to try adapting this for the raspberry pi zero ?

### DISCUSSION THREAD

**Discussion by louisreine** [fayaaz/mixxx-pi-gen#60] [Ideas]
Everything is in the title. I'd like a super portative setup, with cheap hardware and a small LCD screen. The idea is to always have a backup even if someone spills its drink on the cdjs ( 💯 )

**Comment by fayaaz** [comment]
Do you use the same images for the zero 2w as the pi3/4? I would like to try this but I don't have one - I have been trying to source one for another project for a while.

**Comment by fayaaz** [comment]
Just tested this on  a zero 2w and it boots completely fine. The 512 mb of RAM is a bit limiting - Mixxx wiki says it requires 1gb minimum.

Maybe the zero 2w is more suited to running something like PiDeck.
